README.md

```diff
@@ -0,0 +1,274 @@
+    dcos config set core.reporting false
+    dcos config set core.timeout 5
+    deactivate
+    cd -
+
+**CLI_TEST_DCOS_URL**: Holds the URL or IP address of the cluster you
+are testing against. If you used `dcos-launch` to launch the cluster, you
+can get the IP of the cluster by running `dcos-launch describe`.
+
+**CLI_TEST_SSH_KEY_PATH**: Points to a private key file used to ssh
+into nodes on your cluster. If you used `dcos-launch` to launch the
+cluster, then this should point to the same file used in your
+`dcos-launch` config. This is used by the node integration tests.
+
+**CLI_TEST_SSH_USER**: Holds the username used to ssh into nodes on
+your cluster. If you used `dcos-launch` with the configuration listed
+above to launch your cluster, then you *must* set this to centos. This
+is used by the node integration tests.
+
+### Running
+
+Now that your environment is set up appropriately, we can start running
+the tests. We have tests both in the `dcos` package (`python/lib/dcos`) and
+in the `dcoscli` package (`python/lib/dcoscli`).
+
+When running the tests, change your current directory to one of those
+two locations and follow the instructions below.
+
+*NOTE*: You **must** have your virtualenv *deactivated* in order to run
+the tests via the commands below. This is very important and often a
+point of much confusion.
+
+If you want to run the full test suite simply run:
+
+    make test
+
+If you want to run only unit tests that match a specific pattern run:
+
+    env/bin/tox -e py35-unit /<test-file>.py -- -k <test-pattern>
+
+If you want to run only integration tests that match a specific pattern
+run:
+
+    env/bin/tox -e py35-integration /<test-file>.py -- -k <test-pattern>
+
+### Other Useful Commands
+
+1.  List all of the supported test environments:
+
+        env/bin/tox --listenvs
+
+2.  Run a specific set of tests:
+
+        env/bin/tox -e <testenv>
+
+3.  Run a specific unit test module:
+
+        env/bin/tox -e py35-unit /<test-file>.py
+
+4.  Run a specific integration test module:
+
+        env/bin/tox -e py35-integration /<test-file>.py
+
+### Releasing
+
+Releasing a new version of the DC/OS CLI is only possible through an
+[automated TeamCity
+build](https://teamcity.mesosphere.io/viewType.html?buildTypeId=DcosIo_DcosCli_Release)
+which is triggered automatically when a new tag is added.
+
+The tag is used as the version number and must adhere to the
+conventional [PEP-440 version
+scheme](https://www.python.org/dev/peps/pep-0440/).
+
+The automated build starts up three jobs to build the platform dependent
+executables (in Windows, OS X, and Linux).
+
+The executables are pushed to s3 and available at
+<https://downloads.dcos.io/binaries/cli/>\<platform\>/x86-64/\<tag\>/dcos.
+The links to each of the platform executables and the release notes are
+published at: <https://github.com/dcos/dcos-cli/releases/tag/>\<tag\>
+
+The automated build also publishes two packages to PyPI using the
```

Comment:
We don't do this anymore, could this line and what's after be removed? https://github.com/mesosphere/install-scripts is also not reachable.